lib/broadway/noop_acknowledger.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Broadway.NoopAcknowledger do
+  @moduledoc false
```

Comment:
Let's make it public?

```suggestion
  @moduledoc """
  An acknowledger that does nothing.
  
  Set automatically on messages that have been acked immediately
  via `Broadway.Message.ack_immediately/1`.
  """
```

Reply:
Good call, didn't notice that the `CallerAcknowledger` is also public.